### PR TITLE
Chore/changelog

### PR DESCRIPTION
**Pull request by ryami333**
Adds an entry to the CHANGELOG for v3, including an upgrade guide. Note also some prettier formatting in all markdown files.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -39,3 +38,39 @@ npm version [TYPE]
...
+The following principles should help to inform decisions at every level of the
+project's architecture, from communication to implementation, and everything in
+between.
+
+## Inclusivity
+
+The driving principal behind the existence of React Accessible Accordion is an
+inclusive web. This most obviously relates to the inclusivity of those with
+disabilities or impairments which require the use of assistive technology, such
+as screenreaders. It should extend though to technological inclusivity, such
```
Comment (samanthaksanders) [inline CONTRIBUTING.md]: This sentence I had to read a few times. I would say something more like:
This inclusivity should also extend to the technical community, so RAA  _tries_ to remain as technologically agnostic as possible.

Or something like that anyway?

Comment (ryami333) [reply CONTRIBUTING.md]: Hah, yes this is an awkward sentence huh??

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,196 @@
...
+1. Rename all of your `AccordionItemTitle` components to `AccordionItemHeading`
+   and then nest an `AccordionItemButton` directly inside of each one. Note that
+   in order for your Accordion to remain spec-compliant, you may _not_ put any
+   other children inside `AccordionItemHeading`.
+
+    ```tsx
+    // Before:
+    import { AccordionItemTitle } from 'react-accessible-accordion';
+
+    const headingBefore = <AccordionItemHeading>Foo</AccordionItemHeading>;
```
Comment (catepalmer) [inline CHANGELOG.md]: Should this be `const headingBefore = <AccordionItemTitle>Foo</AccordionItemTitle>;`?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,7 +1,196 @@
...
+   have a use-case which calls for the ability to manually control expanded
+   state.
+
+    ```tsx
+    // Before
+    import { Accordion, AccordionItem } from 'react-accessible-accordion';
+
+    const accordionBefore = (
+        <Accordion>
+            <AccordionItem expanded />
```
Comment (catepalmer) [inline CHANGELOG.md]: Should the `expanded` prop here have a value?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,7 +1,196 @@
...
+    import { Accordion } from 'react-accessible-accordion';
+
+    const accordionBefore = <Accordion accordion={false} />;
+    ```
+
+    ```tsx
+    // After:
+    import { Accordion } from 'react-accessible-accordion';
+
+    const accordionAfter = <Accordion allowMultipleExpanded />;
```
Comment (catepalmer) [inline CHANGELOG.md]: Should the `allowMultipleExpanded` prop here have a value, i.e. `allowMultipleExpanded={true}`?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,7 +1,196 @@
...
+
+6. Upgrade to React v16.3+
+
+7. Remove your `minimal-example.css` import. These styles only applied
+   `display: none` to panels when collapsed, but browsers apply these styles to
+   elements with the `hidden` attribute, which the `AccordionItemPanel`
+   component now has (when collapsed).
+
+### Added
+
```
Comment (catepalmer) [inline CHANGELOG.md]: Should `AccordionItemState` be listed here as a component that's been added?

Comment (ryami333) [reply CHANGELOG.md]: Yip - will add

File: CHANGELOG.md (hunk 4 above)
Comment (ryami333) [reply CHANGELOG.md]: Not out of necessity, no. This is an equivalent shorthand.

File: CHANGELOG.md (hunk 2 above)
Comment (ryami333) [reply CHANGELOG.md]: Yep, nice catch.

**Comment by ryami333** [comment]
Feedback addressed, so going to go ahead and merge. But feel free to add more comments to the [`next` PR](https://github.com/springload/react-accessible-accordion/pull/188) if you have any. Cheers!

**Pull request merged**